src/plugins/reporter-print.c

```diff
@@ -80,7 +80,7 @@ int main(int argc, char **argv)
         /* We used freopen to change stdout,
          * but libreport_ask() writes to stdout. Can't use that trick anymore.
          */
-        char *msg = NULL;
+        g_autofree char *msg = NULL;
```

Comment:
Oh, for some reason I thought `g_autofree` takes care of freeing before reassignment too. Thanks.

Reply:
The deal with `g_autofree` and friends is that, where supported (it depends on compiler extensions, but I think only MSVC does not have that implemented out the major ones), it results in cleanup being performed when the variables goes out of scope, so you would need to plaster the function with compound statements if you want to avoid ever having to manually free anything.